### DISCUSSION TITLE
Can using multiple processes of this on same pc can disable it to use the api?

### DISCUSSION THREAD

**Discussion by anmol-kmr** [DavidMGilbert/btc-hack#35] [Q&A]
Can using multiple processes of this on same pc can cause it not being able to use the api?
Can using multiple processes of this on same pc can disable it to use the api?

**Comment by MvdS74** [comment]
Even one can cause the API to block you. The description states that it uses the sochain.com api but it does not. (most likely because that one uses a paid api key) It uses the (free) blockcypher api which has a limitation of 100 calls per hour which comes down to one every 36 seconds. However, this script has no error handling in it, so even though it says it is still checking balances, it is actually not because the API has already blocked you after a few seconds.
